_locales/ar/messages.json

```diff
@@ -84,7 +84,7 @@
     "description": "Checkbox label text for decision confirmations"
   },
   "check_selected_items": {
-    "message": "أشّر العناصر المحددة",
+    "message": "افحص العناصر المحددة",
```

Comment:
Wrong. Check context please. This is to "mark" the items not "check their conditions"!